web/package.json

```diff
@@ -29,5 +29,8 @@
     "check": "svelte-check --tsconfig ./jsconfig.json",
     "check:watch": "svelte-check --tsconfig ./jsconfig.json --watch",
     "lint": "eslint --ignore-path .gitignore ."
+  },
+  "dependencies": {
```

Comment:
devDependency... I think.

Reply:
I looked into this a bit - i think since this will be deployed it is needed in dependency
https://stackoverflow.com/questions/39716447/dev-dependencies-vs-dependencies-in-node-js

Reply:
I don't think so. All our Svelte dependencies are devDependencies, because Svelte is *compiled* to Javascript rather than it being a Javascript library, so the resulting Javascript does not actually have any knowledge of Svelte or of any of the libraries.

You can see that so far we have zero dependencies, it is all devDependencies. I'd like to keep this clean like this, if possible. If you *tested* and it doesn't work, fine, but I am pretty sure it should work.